modules/post-installation-adding-nutanix-failure-domains-compute-machines.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-compute-machines_{context}"]
+= Distributing compute machines across failure domains
+
+You can distribute compute machines across Nutanix failure domains by:
+
+* Modifying existing compute machine sets.
+* Creating new compute machine sets.
+
+The following procedure details how to distribute compute machines across failure domains by modifying existing control plane machines sets. For more information on creating a control plane machine set, see "Additional resources".
+
+.Prerequisites
+
+* You have configured the failure domains in the cluster's Infrastructure custom resource definition (CR).
```

Comment:
Remove "definition". Should be "in the cluster's Infrastructure custom resource (CR)".

Reply:
> Remove "definition". Should be "in the cluster's Infrastructure custom resource (CR)".

I am going to check with @jeana-redhat when she returns from PTO next week. I have seen them refer to this as "infrastructure custom resource definition" elsewhere [1] . Additionally - existing published docs [2] (see step 1) refer to `infrastructures.config.openshift.io` as "custom resource definition" (CRD). I will close on this terminology and update as required.

[1] https://github.com/openshift/openshift-docs/pull/70056/files#diff-1f902b9f28f2684b458efb2ab87443a9c36f86c8b5caffea4783dc122bb52183R13
[2] https://55526--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/post-install-vsphere-zones-regions-configuration#specifying-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration

Reply:
@JoelSpeed Can yo clarify here: should "infrastructure custom resource definition (CRD)" or "infrastructure custom resource (CR)" be used here? I think it should be CR without "definition", since CR refers to the custom resource object.

Reply:
It should be `cutom resource (CR)` to refer to the single instance of the custom resource

Reply:
I copied the use in my PR from this PR :) I will update mine accordingly to remove "definition"

Reply:
Also done.